src/Compilers/CSharp/Portable/Symbols/ExtendedErrorTypeSymbol.cs

```diff
@@ -217,7 +219,7 @@ internal override ImmutableArray<NamedTypeSymbol> GetDeclaredInterfaces(ConsList
         /// However, the real definition is in this class so that it can access the private field 
         /// nonErrorGuessType.
         /// </remarks>
-        internal static TypeSymbol ExtractNonErrorType(TypeSymbol oldSymbol)
+        internal static TypeSymbol? ExtractNonErrorType(TypeSymbol oldSymbol)
```

Comment:
>oldSymbol [](start = 67, length = 9)

nullable?

Reply:
@sharwell making sure you see this one.

---
In reply to: [364483268](https://github.com/dotnet/roslyn/pull/39192#discussion_r364483268) [](ancestors = 364483268)

Reply:
Updated